### ISSUE TITLE
Embedding with AzureOpenAI

### ISSUE THREAD

**Issue by nanotronic** [lgrammel/modelfusion#194]
I am trying to get the pdf-chat-terminal example working with Azure.
I have created two api configurations for this. One for embedding and the other for chatting. It looks like this:
```
const embeddApi = new AzureOpenAIApiConfiguration({
  // apiKey: automatically uses process.env.AZURE_OPENAI_API_KEY,
  resourceName: "my-resource-name",
  deploymentId: "my-embedd-id",
  apiVersion: "my-api-version",
})
```
Unfortunately I get the following error `Too many inputs. The max number of inputs is 16...`.

Is there a possibility to set this parameter?

Thanks in advance

**Comment by nanotronic** [comment]
Yes, I will.

**Comment by lgrammel** [comment]
PR included in `v0.89.0`. Thanks!
